encord/storage.py

```diff
@@ -692,6 +692,136 @@ def upload_audio(
         else:
             return upload_result.items_with_names[0].item_uuid
 
+    def upload_text(
+        self,
+        file_path: Union[Path, str],
+        title: Optional[str] = None,
+        client_metadata: Optional[Dict[str, Any]] = None,
+        cloud_upload_settings: CloudUploadSettings = CloudUploadSettings(),
+    ) -> UUID:  # TODO this should return an item?
```

Comment:
This should A) follow convention
B) Outside of convention, returning a UUID is reasonable given that the method is long-polling to a degree.